core/src/main/resources/defaults/blueprints/cdp-data-mart-active-710.bp

```diff
@@ -0,0 +1,200 @@
+{
+  "description": "CDP 1.1 Active Data Mart template with Apache Impala, Hue, Kudu, and Spark",
+  "blueprint": {
+    "cdhVersion": "7.1.0",
+    "displayName": "datamart-active",
+    "services": [
+      {
+        "refName": "hdfs",
```

Comment:
Why do we need HDFS in this template?

Reply:
For the same reason there's HDFS in the existing data mart template: CM apparently can't yet provision a cluster without an HDFS service. See OPSAPS-50807 for details.